google/resource_container_node_pool.go

```diff
@@ -129,10 +180,70 @@ func resourceContainerNodePoolRead(d *schema.ResourceData, meta interface{}) err
 
 	d.Set("name", nodePool.Name)
 	d.Set("initial_node_count", nodePool.InitialNodeCount)
+	if nodePool.Autoscaling != nil && nodePool.Autoscaling.Enabled {
```

Comment:
How do we detect if autoscaling has been turned on out of band and needs to be switched back on?